src/confscreen.cpp

```diff
@@ -218,6 +224,10 @@ void TextWindow::ShowConfiguration() {
             CO(SS.lightDir[i]), i, &ScreenChangeLightDirection,
             SS.lightIntensity[i], i, &ScreenChangeLightIntensity);
     }
+    Printf(false, "%Bp         Ambient Lighting     "
```

Comment:
Actually, this should be lowercase to follow the style of that dialog.
```suggestion
    Printf(false, "%Bp         ambient lighting     "
```

Reply:
Fixed. Good call it's all lower case.